src/components/common/bullets-colin-link.vue

```diff
@@ -0,0 +1,212 @@
+          <v-row>
+            <slot name="name-input-slot">Name Input</slot>
+            <template v-if="showDesignation">
+              <slot name="designation">Designation</slot>
+            </template>
+          </v-row>
+        </div>
+        <div v-else>
+          <ul class="bullet-points">
+            <li v-for="bulletPoint in bulletPoints" :key="bulletPoint">
+              {{ bulletPoint }}
+            </li>
+          </ul>
+          <div class="btn-spacing" v-if="colinButton">
+            <v-btn class="px-9" :href="colinLink" target="_blank">
+              Go to Corporate Online to Register <v-icon small class="ml-1">mdi-open-in-new</v-icon>
+            </v-btn>
+          </div>
+          <div class="btn-spacing" v-else>
+            <v-btn
+              class="px-9"
+              @click="incorporateNow"
+            >
+              Incorporate Now
+            </v-btn>
+          </div>
+        </div>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { CompanyType } from '@/enums'
+import NameInput from '@/components/new-request/name-input.vue'
+import { Navigate } from '@/plugins'
+import BusinessServices from '@/services/business.services'
+import { BusinessRequest } from '@/interfaces'
+
+@Component({
+  components: {
+    NameInput
+  }
+})
+export default class BulletsColinLink extends Vue {
+  /** The selected business type. */
+  @Prop({ default: '' }) readonly businessType!: string
+
+  /** Show Colin Button. */
+  @Prop({ default: false }) readonly colinButton!: boolean
+
+  /** Show Designation Dropdown. */
+  @Prop({ default: false }) readonly showDesignation!: boolean
+
+  /** Whether user is authenticated. */
+  @Getter getIsAuthenticated!: boolean
+
+  // Local properties
+  selectedCompanyType = ''
+  bulletPoints = [
+    'Your business name will be the Incorporation Number assinged by the Registry.',
+    'You can change your business name at a later date.',
+    'It is not possible to request a specific Incorporation Number.'
+  ]
+  readonly colinLink = sessionStorage.getItem('CORPORATE_ONLINE_URL')
+
+  // For template
+  readonly CompanyType = CompanyType
+
+  // Get radio text respective to Enum values of CompanyType.
+  getRadioText (option: string): string {
+    if (option === CompanyType.NAMED_COMPANY) {
+      return `Named Company`
+    } else if (option === CompanyType.NUMBERED_COMPANY) {
+      return 'Numbered Company'
+    }
+    return '[error]'
+  }
+
+  /**
+   * The alternate short names for entity types.
+   * Alternate names are used in Entities UIs.
+   */
+  entityTypeAlternateShort (entityType: string): string {
+    if (entityType === 'BC' || entityType === 'CC') return entityType
+    if (entityType === 'CR') return 'BEN'
+    if (entityType === 'UL') return 'ULC'
+    return ''
```

Comment:
I think this would be cleaner and more extensible as a switch-case statement.

If the entity type does not need to be overridden then I would just return it (instead of specifically handling BC and CC or returning empty string).